astro/src/content/json/features.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "displayName": "Advanced and Gaming Identity Providers",
```

Comment:
Despite our docs stating that this is for Essentials, I found that we made this available to all licensed editions. Updated this accordingly.

Reply:
I think you mean all paid editions, right?

Reply:
Did you check the identity provider pages to make sure the correct blurb is shown?

Reply:
That right, all paid editions now that we have Licensed Community. Starter plans were also getting this, despite what our docs were saying.